app/services/autoscale-formula.service.ts

```diff
@@ -0,0 +1,72 @@
+import { Injectable } from "@angular/core";
+import { AutoscaleFormula } from "app/models";
+import { Constants, log } from "app/utils";
+import * as storage from "electron-json-storage";
```

Comment:
I made a service wrapping around `electron-json-storage` called `LocalFileStorage`. Use that instead it makes the testing easier and it is also cleaner.